README.md

```diff
@@ -32,8 +32,48 @@ dependencies {
 ```
 <!-- x-release-please-end -->
 
+
+## Releases
+
+The Path SDK is published via [JitPack](https://jitpack.io/#mxenabled/path-sdk).
+
+### Branching Strategy
+
+#### master
+
+`master` contains the edge development for the current major version. All code merged into `master` should be complete, tested, and releasable (from a feature branch).
+
+#### feature branches
+
+  1. Developers create feature branches to introduce code changes
+  2. When complete, a Pull Request is created from the branch
+  3. Once approved, the pull request is merged into `master`
+
+#### release branches
+
+When work on a _planned_ major version is ready to start, a release branch will be created to hold the in-progress the work. Multiple feature branches and pull requests will be generated against this branch until it the version is ready for release.
+
+### Release versioning
+
+We use semantic versioning. git tags are placed at the point (SHA) where the version was generated.
+
+_Current Major Version_: On the _current major version_, minor and patch versions can be generated at any time and are expected to be backward compatible.
+
+_Past Major Versions_: If a severe issue is discovered in a past major version, a release branch will be created off of the last patch version and a service pack will be released (example: 2.3.2-sp.1) from that branch.
+
+_Next Major Version_: Major version releases can come follow 2 different paths:
+
+  1. Planned (typical): When a large upgrade is planned, a release branch will be generated to hold the new code away from master as it is built.
+  2. Unplanned: Some breaking changes may a). be deemed necessary for the health of the project or b). be of minor impact to users of the library. These changes will follow the normal flow and will be merged directly to _master_ (from feature branches).
```

Comment:
```suggestion
  2. Unplanned: Some breaking changes may a) be deemed necessary for the health of the project or b) be of minor impact to users of the library. These changes will follow the normal flow and will be merged directly to _master_ (from feature branches).
```
Suggestion. I think the extra periods look weird.

Reply:
Fixed.